### PR TITLE
Introduce a jar task to build without adventure dependencies

### PR DESCRIPTION
**Pull request by AbhigyaKrishna**
Command: `.\gradlew build -Pno-adv`

Potential solution: #974

### CODE REVIEW

File: buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts [hunk 1]
```diff
@@ -20,9 +20,45 @@ tasks {
...
+        val sourceSets = project.extensions.getByType<SourceSetContainer>()
+
+        manifest.inheritFrom(shadowJar.manifest)
+
+        from(sourceSets.main.get().output)
+        configurations = shadowJar.configurations
+
+        relocate("net.kyori.adventure.text.serializer", "io.github.retrooper.packetevents.adventure.serializer")
+        relocate("net.kyori.option", "io.github.retrooper.packetevents.adventure.option")
+
```
Comment (booky10) [inline buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts:39]: `relocate("org.bstats", "io.github.retrooper.packetevents.bstats")`

The bstats relocation is missing here

**Review by booky10 — COMMENTED** [review]
I've found one minor issue, looks good otherwise :+1:

File: buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts (hunk 1 above)
Comment (retrooper) [reply buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts:39]: Has this been resolved?

Comment (booky10) [reply buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts:39]: no, not yet

Comment (AbhigyaKrishna) [reply buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts:39]: It should be added after the merging of this pr to preserve a better commit history

**Comment by retrooper** [comment]
Thank you for the PR, let's look into the bstats relocation.

**Pull request merged**